scripts/devicetypes.py

```diff
@@ -0,0 +1,405 @@
+
+"""
+DeviceType Library Management:
+Allows:
+
+- Import DeviceTypes Library from repo
+- Update repo (via pull request)
+- Import by Manufacturer
+- Import by Model (filename)
+
+Inspired by https://github.com/netbox-community/reports/pull/34
+Requires: GitPython
+
+Installation:
+1. Copy to the scripts dir
+2. Set up parent dir (user must have write permissions within this dir)
```

Comment:
Change "Set up the parent dir" to "Create a new empty folder and update the `devicetype_parent_dir` variable to reference the folder.

Reply:
[Fixed](https://github.com/Dzordzu/reports/commit/066fa1096074797b0ee7f920e7ede92e839dea8c)